specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/machineLearningServices.json

```diff
@@ -349,6 +349,48 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/notebookAccessToken": {
```

Comment:
>notebookAccessToken [](start = 143, length = 19)

The name of POST action should be imperative (should include a verb). In addition, if you named it something like listAccessToken or listNotebookAccessToken, it would be possible to retrieve the token in an ARM template, for example. (ARM templates expose such actions as list* functions)

See [https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-resource#list](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-resource#list) for details.

Reply:
great! will fix that

Reply:
Thanks for fixing.

---
In reply to: [515323749](https://github.com/Azure/azure-rest-api-specs/pull/11355#discussion_r515323749) [](ancestors = 515323749)